src/client/components/EmailSentInformationBox.tsx

```diff
@@ -89,12 +91,20 @@ export const EmailSentInformationBox = ({
 				)}
 				{changeEmailPage && (
 					<>
-						, or{' '}
+						,{!showSignInWithPasswordOption ? <> or </> : <> </>}
```

Comment:
sorry I'm having a bit of trouble understanding the intent (or maybe the syntax) here:

is the intention that if it has to show both links it would say 
`try another address or sign in with a password instead`
but if it only has to one it would just say `or try another address / or sign in with a password instead` ?

Reply:
the latter option!

Reply:
you can see in storybook on how it behaves: https://60929d168594f80039336501-dlorewwtdy.chromatic.com/?path=/story/components-emailsentinformationbox--with-show-sign-in-with-password-option&globals=viewport:mobile

Reply:
is a comma before 'or' correct ?
It looks weird to me but sadly I don't have an english degree to know this kind of things for sure

Reply:
In generally I've always used a comma before the or in the last item in a list, so it separated from the other items in the same way, like an array.

See `Oxford comma` in the Guardian style guide: https://www.theguardian.com/guardian-observer-style-guide-o#:~:text=what%20we%20use-,Oxford%20comma,-a%20comma%20before